.github/workflows/gohai.yml

```diff
@@ -0,0 +1,24 @@
+name: "Gohai Test"
+
+# Only run the tests if pkg/gohai was changed
+on:
+  push:
+    paths:
+      - 'pkg/gohai/**'
+  pull_request:
+    paths:
+      - 'pkg/gohai/**'
```

Comment:
Is it needed to run these on all pushes and all pull requests? This makes them run twice per PR, which is a bit wasteful.

You could follow what is done for other workflows in this repositories, and run this on all pull requests, and on pushes to main and release branches ([example](https://github.com/DataDog/datadog-agent/blob/ce2cdc683c9a5e4f7240d1c99fc620a784dc0e59/.github/workflows/windows-lint-go.yml#L4-L8)).